src/interceptors/add_idp_client_id_to_params.js

```diff
@@ -0,0 +1,17 @@
+/**
+   Read `idpClientId` from `ctx` and add it to `params`
+
+   Changes to `ctx`:
+
+   - add `params.idpClientId`
+ */
+export default class AddClientIdToParams {
```

Comment:
This could be named `AddIdpClientIdToParams`, even though it seems to work when imported as that in sdk.js.

Reply:
Oops, I think this is just copy paste error and it should be `AddIdpClientIdToParams`. I'm going to change that!